cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/ScheduleTaskRunner.java

```diff
@@ -138,4 +140,16 @@ public void execute(ProgramId id, Map<String, String> sysArgs, Map<String, Strin
       SecurityRequestContext.setUserId(originalUserId);
     }
   }
+
+  /**
+   * Get the latest version for a program.
+   */
+  private ProgramId getLatestProgramId(ProgramId programId) {
```

Comment:
nit: This should be `getLatestAppProgramId` ? since latest is for the app and not the the program?

Reply:
agree, fixed